modules_core/com.smf.jobs.defaults/src/com/smf/jobs/defaults/Post.java

```diff
@@ -128,9 +131,32 @@ private String getDateProperty(String adTableId, VariablesSecureApp vars) {
 
     String property = mapOfTablesSupported.get(adTableId);
     if (property == null) {
-      throw new OBException(OBMessageUtils.messageBD(new DalConnectionProvider(), "TableNotFound", vars.getLanguage()));
+      /*
+       * Position 0 = ACCTDATECOLUMN
+       * Position 1 = ACCTCLASSNAME
+       * */
+      Object[] acctinfo = getTableInfo(adTableId);
+      if (acctinfo == null) {
+        throw new OBException(
+            OBMessageUtils.messageBD(new DalConnectionProvider(), "TableNotFound", vars.getLanguage()));
+      }
+      if (acctinfo.length != 0 && (!acctinfo[1].equals("") && !acctinfo[0].equals(""))) {
```

Comment:
Will be more accurate if you check if array length is equal to 2 if you later in condition you compare index 0 and 1 with an empty string.